01.md

```diff
@@ -92,7 +92,7 @@ Kinds specify how clients should interpret the meaning of each event and the oth
 
 And also a convention for kind ranges that allow for easier experimentation and flexibility of relay implementation:
 
-- for kind `n` such that `1000 <= n < 10000`, events are **regular**, which means they're all expected to be stored by relays.
+- for kind `n` such that `1000 <= n < 10000 || 4 <= n < 45 || n == 1 || n == 2`, events are **regular**, which means they're all expected to be stored by relays.
```

Comment:
`kind: 41` is a replaceable event according to NIP-28 and nostr-rs-relay.

https://github.com/nostr-protocol/nips/blob/1728f93d17fe399fa6e0911e7bb69d2e159ca572/28.md?plain=1#L40

https://github.com/scsibug/nostr-rs-relay/blob/4f518fd0e765d20a8a8d07bc0a17ef9fbb7bfbb6/src/event.rs#L171-L178

Reply:
Thanks for your response, I went through all the other `kind`s and this seems to be the only one mentioning a different lifetime. I now included kind 41.

Looking at kind 40 this leaves me wondering about its purpose. Why does kind 40 (create channel) contain the same metadata that can be modified in kind 41 (set metadata)? Doesn't it make more sense to have just kind 41 fullfill both purposes similar to how [NIP-52](https://github.com/nostr-protocol/nips/blob/master/52.md#format) (calendar event, kind 31923) is set up?

**Couple other points:**
- Just to verify: kind 40 is just of lifecycle regular, correct?
- I don't see any definition of kind 33. But it looks like deleted one given your [recent PR](https://github.com/nostr-protocol/nips/pull/1318).